cmd/cstor-pool-mgmt/controller/replica-controller/handler.go

```diff
@@ -379,11 +379,32 @@ func (c *CStorVolumeReplicaController) getVolumeReplicaResource(
 	return cStorVolumeReplicaUpdated, nil
 }
 
+/*
+func PoolName(cvr *apis.CStorVolumeReplica) string {
+	if string(cvr.ObjectMeta.Labels["cstorpool.openebs.io/uid"]) != "" {
+		return cvr.Labels["cstorpool.openebs.io/uid"]
+	}
+	if string(cvr.ObjectMeta.Labels["cstorpoolinstance.openebs.io/uid"]) != "" {
+		return os.Getenv(string(common.OpenEBSIOPoolName))
+	}
+	return ""
+}
+*/
+
 // IsRightCStorVolumeReplica is to check if the cvr
 // request is for particular pod/application.
 func IsRightCStorVolumeReplica(cVR *apis.CStorVolumeReplica) bool {
-	if os.Getenv(string(common.OpenEBSIOCStorID)) == string(cVR.ObjectMeta.Labels["cstorpool.openebs.io/uid"]) {
-		return true
+	if strings.TrimSpace(string(cVR.ObjectMeta.Labels["cstorpool.openebs.io/uid"])) != "" {
+		if os.Getenv(string(common.OpenEBSIOCStorID)) == string(cVR.ObjectMeta.Labels["cstorpool.openebs.io/uid"]) {
+			return true
+		}
+		return false
+	}
```

Comment:
For backup/restore controller, we are using `cvr.ObjectMeta.Labels["cstorpool.openebs.io/uid"]`. With this change, we should use either `cvr.ObjectMeta.Labels["cstorpool.openebs.io/uid"]` or `cvr.ObjectMeta.Labels["cstorpoolinstance.openebs.io/uid"]`, right?

Reply:
Yes , right